cni/log/logger.go

```diff
@@ -0,0 +1,44 @@
+package log
+
+import (
+	"go.uber.org/zap"
+	"go.uber.org/zap/zapcore"
+	"gopkg.in/natefinch/lumberjack.v2"
+)
+
+type Config struct {
+	Level       zapcore.Level
+	LogPath     string
+	MaxSizeInMB int
+	MaxBackups  int
+	Name        string
+}
+
+var Logger *zap.Logger
+
+func New(cfg *Config) (func(), error) {
```

Comment:
can you add brief comment for this function on what it does and what it returns?

Reply:
done